daemon-scheduler/pkg/engine/scheduler.go

```diff
@@ -37,20 +39,20 @@ const (
 )
 
 type scheduler struct {
-	id             string
-	ctx            context.Context
-	environmentSvc deployment.Environment
-	deploymentSvc  deployment.Deployment
-	css            facade.ClusterState
-	ecs            facade.ECS
-	events         chan<- Event
-	executionState map[string]environmentExecutionState
-	inProgress     bool
-	inProgressLock sync.RWMutex
+	id                 string
+	ctx                context.Context
+	environmentService environment.EnvironmentService
```

Comment:
Can we make this `environmentSvc` to keep it consistent with `deploymentSvc` declared below? Or vice versa (use `..Service` for both).